src/main/java/no/nav/pto/veilarbportefolje/postgres/PostgresOpensearchMapper.java

```diff
@@ -140,4 +144,11 @@ private Statsborgerskap getHovedStatsborgetskapMedFulltLandNavn(Statsborgerskap
                 statsborgerskap.getGyldigFra(),
                 statsborgerskap.getGyldigTil());
     }
+
+    public void flettInnAvvik14aVedtak(List<OppfolgingsBruker> brukere) {
+        Map<GjeldendeIdenter, Avvik14aVedtak> avvik14aVedtakList = avvik14aService.hentAvvik(brukere.stream().map(GjeldendeIdenter::of).collect(Collectors.toSet()));
```

Comment:
I think there is one bottleneck here:
- we fetched active `brukere` from db, and then inside `hentAvvik` there is one more fetch of the same users just in different format...I think there is no need to fetch same data from db twice, if different format is necessary you can just convert existing users to that format.

Reply:
there is actually no need at all to fetch users inside `hentAvvik`. All necessary data are inside `brukere` list

Reply:
Diskutert ved pulten.